### ISSUE TITLE
How can i mine? Have 2 VPS and all is running.

### ISSUE THREAD

**Issue by blueavenue** [forknote/forknote#82]
Hello everybody,

my english is not so good. My Question:

How i can start mining with Daemon oder Simplewallet from my local machine?

I've installed all on my two VPS-Servers (2 Core - 4GB) with instruction from forknote.net. All is running and sync. The Services are startet in etc/init. 

**But nothing happens. All Servers and local machine are seems to be waiting
for a command or whatever.** 

When i write --help for the downloadable ./simplewallet in the terminal, there is no command like --start-mining.

When i start the daemon from my local machine ./mycoind --config-file configs/mycoin.conf
all starts, but then nothing happens.

What i have to do to get the coin started and mine?

I have:

2 VPS with 2 Core and 4 GB RAM (Ubuntu 16.04. minimum)
1 Local Machine (With Win 10 and Ubuntu 16.04.3 LTS)
I do all from the documentation and installation instruction from forknote.net

useradd fork, new genesis block etc, mycoin.conf with add seed nodes...

I tried further with Ubunutu 14.04. minimum. But nothing happens.

Another question. How works the ./miner? 

Please help me. Thank you all!

See screens below (first before i start daemon with --log-level 5 and second after --log-level 5)

![problem](https://user-images.githubusercontent.com/35180661/35769193-a453b0da-0907-11e8-81a0-353a2c6ff446.jpg)

![problem2](https://user-images.githubusercontent.com/35180661/35769414-ab7de7a6-090a-11e8-9346-5e21f49f8491.jpg)

**Comment by blueavenue** [comment]
I used upstart in etc/init 

And start forknoted-mycoin-daemon from forknoted-mycoin-daemon.conf (like form from forknote,net with my foldername and coinname. It works, because i sudo reboot the system and tried to start.

The System says: Job already startet (on both VPS). I tried your miner tip (this time with miner.exe not with ./miner, but he does nothing... 
1. I stopped local Daemon
2. I stopped simplewallet
3. I tried miner with local daemon

Nothing... the cmd freeze.

look picture:

![miner](https://user-images.githubusercontent.com/35180661/35770148-5665f568-0916-11e8-9267-7e242f499fae.jpg)
